### PR TITLE
Collected changes for ETC_en.tsv from translator.tosbase.com

### PR DESCRIPTION
**Pull request by TOSBase**

### CODE REVIEW

File: ETC_en.tsv [hunk 1]
```diff
@@ -928,8 +928,8 @@ ETC_20150317_000927	Mushwort
 ETC_20150317_000928	Q3 Mushcaria
 ETC_20150317_000929	Q1 Mothstem
 ETC_20150317_000930	Q1 Moa
-ETC_20150317_000931	Q1 Minotaurss
-ETC_20150317_000932	Q2 Minotaurss
+ETC_20150317_000931	Q1 Minotaurs
```
Comment (Sunaries) [inline ETC_en.tsv]: Minotaurs -> Minotaur

File: ETC_en.tsv [hunk 2]
```diff
@@ -928,8 +928,8 @@ ETC_20150317_000927	Mushwort
...
+ETC_20150317_000932	Q2 Minotaurs
```
Comment (Sunaries) [inline ETC_en.tsv]: Minotaurs -> Minotaur

File: ETC_en.tsv [hunk 3]
```diff
@@ -6304,19 +6304,19 @@ ETC_20150317_006303	Clerics heal and protect allies through the blessings of the
...
 ETC_20150317_006312	Sadhu can use their soul to disappear from the enemies or convey their powers to other players.
 ETC_20150317_006313	Dievdirby
-ETC_20150317_006314	Dievdirbys carve and place the holy statues. The statues protect allies or attack the enemies.
+ETC_20150317_006314	Dievdirbys carve and place holy statues which protect allies or attack enemies.
 ETC_20150317_006315	Oracle
 ETC_20150317_006316	Monk
 ETC_20150317_006317	Monks are clerics who train in martial arts through the blessing of the Goddess.
 ETC_20150317_006318	Pardoner
-ETC_20150317_006319	Pardoners are clerics who don't deny to be blessed by the Goddess with their wealth.
+ETC_20150317_006319	Pardoners are clerics who don't deny to increse their wealth by the Goddesses' blessing.
```
Comment (Sunaries) [inline ETC_en.tsv]: Goddesses' -> Goddess's

File: ETC_en.tsv [hunk 4]
```diff
@@ -11504,7 +11504,7 @@ ETC_20150428_011507	Observing the shattered statue
 ETC_20150428_011508	This reagent seems not made properly!{nl}Tell Grachius about this fact
 ETC_20150428_011509	It seems to be returning to the original sanctuary.{nl}Fine the Master Priest.
 ETC_20150428_011510	The seal has been released!
-ETC_20150428_011511	It is unable to unseal since there are no keys of great king.
+ETC_20150428_011511	Unable to unseal because you don't have a great key.
```
Comment (Sunaries) [inline ETC_en.tsv]: "_You are unable to unseal because you don't have the Great King's key._"

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**